virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc

```diff
@@ -1,14 +1,14 @@
 :_content-type: ASSEMBLY
 [id="virt-using-the-default-pod-network-with-virt"]
-= Using the default pod network for virtual machines
+= Configuring the default pod network
 include::_attributes/common-attributes.adoc[]
 :context: virt-using-the-default-pod-network-with-virt
 
 toc::[]
 
 You can use the default pod network with {VirtProductName}. To do so, you must use the `masquerade` binding method. Do not use `masquerade` mode with non-default networks.
 
-xref:../../../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#attaching-to-multiple-networks[For secondary networks], use the `bridge` binding method.
+xref:../../../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-attaching-vm-multiple-networks[For external networks], use the `bridge` binding method.
```

Comment:
I would remove this. It's irrelevant for this assembly and the link will become invalid in 4.12.

Reply:
Axed that xref.